mesonbuild/linkers.py

```diff
@@ -763,33 +781,36 @@ def get_debug_crt_args(self) -> typing.List[str]:
         MSVC won't auto-select a CRT for us in that case and will error out
         asking us to select one.
         """
-        return ['/MDd']
+        return self._apply_prefix('/MDd')
 
     def get_output_args(self, outputname: str) -> typing.List[str]:
-        return ['/MACHINE:' + self.machine, '/OUT:' + outputname]
+        return self._apply_prefix('/MACHINE:' + self.machine) + self._apply_prefix('/OUT:' + outputname)
 
     def get_always_args(self) -> typing.List[str]:
-        return ['/nologo']
+        return self._apply_prefix('/nologo')
 
     def get_search_args(self, dirname: str) -> typing.List[str]:
-        return ['/LIBPATH:' + dirname]
+        return self._apply_prefix('/LIBPATH:' + dirname)
 
     def get_std_shared_lib_args(self) -> typing.List[str]:
-        return ['/DLL']
+        return self._apply_prefix('/DLL')
 
     def get_debugfile_args(self, targetfile: str) -> typing.List[str]:
         pdbarr = targetfile.split('.')[:-1]
         pdbarr += ['pdb']
-        return ['/DEBUG', '/PDB:' + '.'.join(pdbarr)]
+        return self._apply_prefix('/DEBUG') + self._apply_prefix('/PDB:' + '.'.join(pdbarr))
 
     def get_link_whole_for(self, args: typing.List[str]) -> typing.List[str]:
         # Only since VS2015
         args = mesonlib.listify(args)
-        return ['/WHOLEARCHIVE:' + x for x in args]
+        l = []  #typing.List[str]
```

Comment:
__[Flake8]__ [E262] inline comment should start with '# ' ([link](https://sider.review/gh/repos/19784232/pulls/5856#issue-4491600))

<sub>posted by Sider</sub>